routes/furniture.js

```diff
@@ -43,23 +36,17 @@ router.post('/', async (req, res) => {
 router.put('/:furnitureId', async (req, res) => {
   try {
     const { furnitureId } = req.params;
-    const { id, donationId, name, description, notes } = req.body;
+    const { donationId, name } = req.body;
     const updatedFurniture = await db.query(
       `UPDATE furniture
          SET
-            id = $(id),
-            donationId = $(donationId),
+            donation_id = $(donationId),
             name = $(name)
```

Comment:
for `put` routes all parameters should be optional, also do an edge case check if nothing is passed in